packages/perseus-editor/src/editor-page.tsx

```diff
@@ -193,13 +193,15 @@ class EditorPage extends React.Component<Props, State> {
         return issues1.concat(issues2);
     }
 
-    serialize(options?: {keepDeletedWidgets?: boolean}): any | PerseusItem {
+    serialize(options?: {keepDeletedWidgets?: boolean}): PerseusItem {
         if (this.props.jsonMode) {
             return this.state.json;
         }
-        return _.extend(this.itemEditor.current?.serialize(options), {
-            hints: this.hintsEditor.current?.serialize(options),
-        });
+        return {
+            ...this.itemEditor.current!.serialize(options),
+            hints: this.hintsEditor.current!.serialize(options) ?? [],
+            answer: undefined,
```

Comment:
What's the purpose of setting `answer` to `undefined` here? We weren't doing that before.

Reply:
I explained above to the same question Nisha posed. It's because the field is defined as `answer: any`, so apparently TypeScript requires it to be present in the returned object even if we specify it's value is `undefined`.